### PR TITLE
Update PullApprove configuration 

### PR DESCRIPTION
**Pull request by aserv92**
Rename the PullApprove configuration. Modify PullApprove conditions and add unmet status explanations.

### CODE REVIEW

**Comment by aserv92** [comment]
> /pullapprove-test
> 
> ```yaml
> version: 3
> extends: "https://api.github.com/repos/medology/Scripts/contents/Pull%20Approve/base.pullapprove.yml?ref=master"
> pullapprove_conditions:
>   - 
>     condition: "base.ref == '2.0'"
>     unmet_status: failure
>     explanation: "Review not required unless merging to master"
> ```

This is the correct answer. New line after the dash

File: .pullapprove.yml [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+  - 
+    condition: "'[DO NOT REVIEW]' not in title"
+    unmet_status: pending
+    explanation: "Not ready for review"
+  - 
+    condition: "state == 'open'"
+    explanation: "PR must be open to be reviewed"
+    unmet_status: failure
+  - 
+    condition: "'ci/circleci: build' in statuses.succeeded or 'Force Review' in labels"
```
Comment (fortoulrodolfo) [inline .pullapprove.yml]: Do we have a `Force Review` label? 🤔

Comment (aserv92) [reply .pullapprove.yml]: Now we do.
The options are either a label or we have to type somthing in the title. Ramsey recommended the label. What do you think?

Comment (fortoulrodolfo) [reply .pullapprove.yml]: Please clarify this with Donald. We maybe need to discuss it as a team.

**Comment by aserv92** [comment]
testing @fortoulrodolfo

File: .pullapprove.yml (hunk 1 above)
Comment (aserv92) [reply .pullapprove.yml]: I created a poll on slack and everyone seems okay with the labeling solution

Screenshot from 2019-09-05 18-06-44

**Pull request merged**